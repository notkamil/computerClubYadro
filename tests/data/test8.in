1
09:00 18:00
10
08:45 1 client_early