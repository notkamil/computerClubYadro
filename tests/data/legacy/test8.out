09:00
08:45 1 client_early
08:45 13 NotOpenYet
18:00
1 0 00:00